src/card.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+export default function Card(props) {
+  return (
+    props.card.length !== 0 ?
```

Comment:
Rather than use a giant ternary expression filled with JSX, break this into two subcomponents. You can still use a ternary, you just want to keep it concise.